.github/workflows/pull-request.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         run: echo "/usr/local/bin" >> $GITHUB_PATH
 
       - name: Checkout
-        uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # pin@v4
```

Comment:
Are the changes to the workflow files intended?

Reply:
They are 🙂 It's because dependabot has opened other PRs for the version changes, but it doesn't open those PRs to conform to conventional commits, which breaks the automated versioning in this repo. Until that gets sorted, we either make a new PR for them or bundle them into an existing one (at least I think that's the reason!)